src/Cli/ConfigGenerator.cs

```diff
@@ -222,9 +222,9 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
 
             if (runtimeBaseRoute is not null)
             {
-                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out EasyAuthType authMode) || authMode is not EasyAuthType.StaticWebApps)
+                if (!Enum.TryParse(options.AuthenticationProvider, ignoreCase: true, out EasyAuthType authMode) || authMode is not EasyAuthType.StaticWebApps or EasyAuthType.EasyAuth or EasyAuthType.None)
                 {
-                    _logger.LogError("Runtime base-route can only be specified when the authentication provider is Static Web Apps.");
+                    _logger.LogError("Runtime base-route can only be specified when the authentication provider is Static Web Apps, EasyAuth or None.");
```

Comment:
Lets also remove `None` as synonym for `StaticWebApp`. I know the issue said to add None, but we really need to verify what if AppService == StaticWebApp and make this decision.